client/src/routes/config.ts

```diff
@@ -1,52 +1,38 @@
-import {
-  Auth,
-  FoundationAnalytics,
-  FoundationAnalyticsEvent,
-  FoundationAnalyticsEventType,
-  Session,
-} from '@genesislcap/foundation-comms';
-import {
-  defaultLoginConfig,
-  LoginConfig,
-  Settings as LoginSettings,
-} from '@genesislcap/foundation-login';
-import { Constructable } from '@microsoft/fast-element';
-import { Container, optional } from '@microsoft/fast-foundation';
-import { Route, RouterConfiguration } from '@microsoft/fast-router';
+import { Auth, Session } from '@genesislcap/foundation-comms';
+import { defaultLoginConfig, LoginConfig, Settings as LoginSettings } from '@genesislcap/foundation-login';
+import { FoundationRouterConfiguration } from '@genesislcap/foundation-ui';
+import { optional } from '@microsoft/fast-foundation';
+import { Route } from '@microsoft/fast-router';
 import { defaultLayout, loginLayout } from '../layouts';
 import { Home } from './home/home';
 import { NotFound } from './not-found/not-found';
 
 // eslint-disable-next-line
 declare var ENABLE_SSO: string;
 
-const ssoSettings =
-  typeof ENABLE_SSO !== 'undefined' && ENABLE_SSO === 'true'
-    ? {
-        autoAuth: true,
-        sso: {
-          toggled: true,
-          identityProvidersPath: 'sso/list',
-        },
-      }
-    : {};
+const ssoSettings = typeof ENABLE_SSO !== 'undefined' && ENABLE_SSO === 'true'
+  ? {
+      autoAuth: true,
+      sso: {
+        toggled: true,
+        identityProvidersPath: 'sso/list',
+      },
+    }
+  : {};
```

Comment:
We'll need to double check these formatting changes - formatting this way, so that linting passes in a freshly generated app.

Reply:
We're actually checking that automatically as part of the build. It was just failing on earlier step, so it'll be flagged up by the PR build as needed.